pindel/pindel.cwl

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env cwl-runner
+
+cwlVersion: v1.0
+class: CommandLineTool
+label: "pindel v0.2.5b8"
+baseCommand: ["/usr/bin/pindel", "-i", "pindel.config"]
+requirements:
+    - class: DockerRequirement
+      dockerPull: "mgibio/pindel-cwl:v0.2.5b8"
+    - class: InitialWorkDirRequirement
+      listing:
+          - entryname: 'pindel.config'
+            entry: |
+                $(inputs.normal_bam.path) 400    normal
```

Comment:
For the sample names, we may want to use all caps `TUMOR` and `NORMAL` to match the other callers.

For the insert size, should this be an optional argument with a default of 400?  For example, the HCC1395 exome data has a median closer to 200bp.